app/services/disability_claim_service.rb

```diff
@@ -33,6 +33,15 @@ def request_decision(id)
     client.submit_5103_waiver(id).body
   end
 
+  # upload file to s3 and enqueue job to upload to EVSS
+  # TODO (AJM): Unique filename in s3 but preserve original filename when uploading to evss
+  # TODO (AJM): Give user GlobalID for serialization
```

Comment:
From what I heard during the eng meeting today, users are not guaranteed to be in Redis after logging out/session expires, so if this job had to be retried later in the day the lookup may fail. Is that correct @saneshark ? If so, it may be easier to just pass the user attributes as a hash and reconstruct the User object in the job, are there any downsides to that approach?